test/TestConfig/settings.mk

```diff
@@ -206,7 +206,11 @@ REPORTDIR = $(Q)$(TESTOUTPUT)$(D)$@$(Q)
 #######################################
 # TEST_STATUS
 #######################################
-TEST_STATUS=if [ $$? -eq 0 ] ; then $(ECHO) $(Q)$(Q); $(ECHO) $(Q)$@$(Q)$(Q)_PASSED$(Q); $(ECHO) $(Q)$(Q); $(CD) $(TEST_ROOT); $(RM) -r $(REPORTDIR); else $(ECHO) $(Q)$(Q); $(ECHO) $(Q)$@$(Q)$(Q)_FAILED$(Q); $(ECHO) $(Q)$(Q); fi
+REPORT_CLEAN_CMD=
+ifeq ($(REPORT_CLEANUP), true)
```

Comment:
Could we have a similar name as `KEEP_WORKSPACE`? 

```
ifeq ($(KEEP_REPORTDIR), false)
...

```

Similarly, could we have `RM_REPORTDIR` instead of `REPORT_CLEAN_CMD`?

Reply:
I will update them. A question, KEEP_REPORTDIR can be true or false by default, which one do you think is better?

Reply:
My preference is to KEEP_REPORTDIR true by default, as that is how it originally was designed, that is what a user running it locally would expect (and its the behaviour other underlying test frameworks also have).  

The main reason we added support to remove the report dir is to alleviate storage issues on Jenkins.  So, for the automated Jenkins jobs we can set KEEP_REPORTDIR to false.

Use case 1: local testing, expectation is keeping report dir (we should keep the number of variables individual devs have to set, so default setting of true makes sense)
Use case 2: Jenkins testing, expectation is removing report dir on successful test run - this is a one time change in scripts (low-cost, low aggravation)

Reply:
Updated